lib/update-check.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var latestVersion = require('latest-version');
+var semver = require('semver');
+var info = require('../package.json');
+var settings = require('../settings');
+var Spinner = require('cli-spinner').Spinner;
+var chalk = require('chalk');
+Spinner.setDefaultSpinnerString(Spinner.spinners[7]);
+
+function check(next) {
+	console.error('particle-cli v' + info.version);
+	console.error();
```

Comment:
I think it would be very helpful when helping troubleshoot to have the version number there. It removes a round trip of us asking for the version. I did have it go to `stderr` instead so that if scripts were processing `stdout` they would not be affected.